docs/contribution_handling.md

```diff
@@ -0,0 +1,53 @@
+# WinUI Repo Contribution Handling
+
+The WinUI repo is intended both as a place for the WinUI community to discuss development with each other, and as a place to provide insight into upcoming changes and bug fixes the WinUI team is working on before updates are released. Here, we’ll outline how we handle feature requests and bugs that the community opens.
+
+## Issues
+
+Feature requests and bugs are tracked as GitHub issues.
+
+For reporting security issues please see the [Security Policy](SECURITY.md).
+
+For all other bugs and general issues please [file a new issue](https://github.com/Microsoft/microsoft-ui-xaml/issues/new/choose) using the Bug Report template.
+
+Please file questions and discussions as discussions, not issues.
+
+## Feature Proposals
+
+Feature proposals for WinUI 3 are categorized based on the likely timeframe in which the feature team can consider them.
+
+1. 	Short-term, less than a year
+2.	Long-term, not likely in the next year
+
+All feature proposals are automatically categorized as long-term and unlikely to be considered, although the WinUI team might change the priority based on customer and business needs. If the WinUI team does close a feature proposal, we will indicate the reason why and how it fits into our plans. Closing a proposal does not mean that the team will never consider it. We encourage the community to add comments or interact with a proposal at any time, even a closed one, to signal its importance to the team.
+
+### Feature Planning
+
+The WinUI team plans on sharing what’s coming in the next major release once the feature list is ready, although due to frequently changing business priorities we are not committed to a long-term roadmap. Additionally, preview and experimental releases provide opportunities for the community to see what’s new and what’s changing on a more frequent basis.
```

Comment:
Catering to developers who wants to build on and support Windows applications should be the main business priority of a developer SDK.

Reply:
It is, and I've updated the wording to be more accurate as to our intentions for the time being. Thanks!